### DISCUSSION TITLE
Preptember and Hacktoberfest

### DISCUSSION THREAD

**Discussion by RMcNeely** [elementary/triage#150] [Ideas]
Friends, Romans, countrymen, it's almost time for Hacktoberfest 2022 (where did the year go?). In that spirit I wanted to ask if there's anything that you might need @danrabbit to help get elementary up and running for the event. I'm just starting a new job but should have my schedule clear up closer to October and can do any sort of reviews for the web. 

P.S. I'm on mobile so if this has been started somewhere else and I missed it, I apologize.

**Comment by vjr** [comment]
Can I suggest that the 7.0 daily ISOs be made available to participants of the hackathon(s) and the images come prebuilt with developer packages such as the elementary SDK flatpak and whatever related packages so that people like me with slow internet do not have to go through long download times preparing our dev environments for making contributions? Regards!
